docs/skopeo-sync.1.md

```diff
@@ -0,0 +1,131 @@
+% skopeo-sync(1)
+
+## NAME
+skopeo\-sync - Synchronize images between container registries and local directories.
+
+
+## SYNOPSIS
+**skopeo sync** [**sync-options**] --src _source-location_ --dest _destination-location_ _source_ _destination_
+
+## DESCRIPTION
+Synchronize images between container registries and local directories.
+The synchronization is obtained by copying all the images found at _source_ to _destination_.
+
+Useful to keep in sync a local container registry mirror, and to to populate registries running inside of air-gapped environments.
+
+Available _source_ locations:
+ - _docker_ (i.e. `--src docker`): _source_ is a repository hosted on a container registry (e.g.: `registry.example.com/busybox`).
+ If no image tag is specified, skopeo sync copies all the tags found in that repository.
+ - _dir_ (i.e. `--src dir`): _source_ is a local directory path (e.g.: `/media/usb/`). Refer to skopeo(1) **dir:**_path_ for the local image format.
+ - _yaml_ (i.e. `--src yaml`): _source_ is local YAML file path.
+ The YAML file should specify the list of images copied from different container registries (local directories are not supported). Refer to EXAMPLES for the file format.
+
+Available _destination_ locations:
+ - _docker_ (i.e. `--src docker`): _destination_ is a container registry (e.g.: `my-registry.local.lan`).
+ - _dir_ (i.e. `--src dir`): _destination_ is a local directory path (e.g.: `/media/usb/`).
+ One directory per source 'image:tag' is created for each copied image.
+
+When the `--scoped` option is specified, images are prefixed at _destination_ with the source image path, so that mutiple images with the same
+name can be contained at _destination_.
```

Comment:
Assuming `--scoped` makes it into the PR: we need to document some shortcoming when using a docker destination. Again the reference parsing is biting us. Assuming the source has a `:port`, it cannot be used with `--scoped`.

Reply:
suggest `s/prefixed at _desitnation_ with the source image path, /with the source image path /` (removed comma too)